app/helpers/application_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 # Rubocop ate my lunch
 module ApplicationHelper
+  include DfE::Autocomplete::ApplicationHelper
```

Comment:
It doesn't look like we're actually using any of the helpers in this module currently?

https://github.com/DFE-Digital/dfe-autocomplete/blob/main/app/helpers/dfe/autocomplete/application_helper.rb#L3